Simplenote/NSBezierPath+Simplenote.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+
+
+// MARK: - NSBezierPath + Simplenote
+//
+extension NSBezierPath {
+
+    /// Initializes a Bezier Path that only renders the specified corners as rounded
+    ///
+    convenience init(roundedRect rect: NSRect, byRoundingCorners corners: RectCorner, radius: CGFloat) {
```

Comment:
Oh, you had to create it yourself. iOS provides this out of the box 🚀

Reply:
Yup... that's AppKit!! 🔥